app/forms/contact/create_project_contact_form.rb

```diff
@@ -66,6 +66,25 @@ def save
     end
   end
 
+  def yes_no_responses
+    [OpenStruct.new(id: true, name: I18n.t("yes")), OpenStruct.new(id: false, name: I18n.t("no"))]
+  end
+
+  private def choose_organisation_name
+    return organisation_name unless organisation_name.blank?
+
+    case category
+    when "school_or_academy"
+      @project.establishment.name
+    when "incoming_trust"
+      @project.incoming_trust.name
+    when "outgoing_trust"
+      @project.outgoing_trust&.name
+    when "local_authority"
+      @project.local_authority&.name
+    end
```

Comment:
Should we default to 'other' if only to keep SonarCloud happy?

Reply:
Unfortunately if we add an else clause here with "nil" for the organisation name (organisation name can be blank), standard.rb removes it as it's redundant! I think we should maybe add an exception is sonarcloud, as there's no reason why we can't have an empty organisation name.

Reply:
SonarCloud is wrong in my view, so happier anyway! 👍